numpy/lib/arraypad.py

```diff
@@ -1428,10 +1452,6 @@ def pad(array, pad_width, mode=None, **kwargs):
                                            pad_iter_a), method, axis)
                 pad_before -= pad_iter_b
                 pad_after -= pad_iter_a
-                if pad_iter_b > 0:
```

Comment:
`offset` was a debug parameter I used to print while tweaking these loops. It was never used in the output. I removed all other instances of it, but this one slipped through resulting in a couple of unnecessary addition operations with `mode='reflect'`.